src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -349,13 +259,15 @@ func (cfg *Config) ApplyDefaults() {
 	}
 
 	if cfg.CloudConfig == "" {
-		cfg.CloudConfig = chainedCloudConfigUrl
+		cfg.CloudConfig = chainedCloudConfigURL
 	}
 
-	if cfg.Client != nil {
-		cfg.applyClientDefaults()
+	if cfg.Client == nil {
+		cfg.Client = &client.ClientConfig{}
```

Comment:
Seems okay.  I think this was a holdover from when flashlight had a server mode.